### DISCUSSION TITLE
Error during hardhat installation.

### DISCUSSION THREAD

**Discussion by itsmeashim** [NomicFoundation/hardhat#3428] [Troubleshooting] [answered]
Error: 
![image](https://user-images.githubusercontent.com/84639398/207677426-0f6e93a0-8de3-4ef6-a81f-800432088688.png)

Node and npm version:
![image](https://user-images.githubusercontent.com/84639398/207677551-417bbb80-6833-4cf2-8b78-ead76e66d33a.png)

Machine details:
![image](https://user-images.githubusercontent.com/84639398/207678051-c1cdf912-688e-4524-991d-ea961dd9666c.png)

**Comment by fvictorio** [comment]
This is super strange, that operator should definitely be supported in the version of node you are using. Please try these two things:

First, open the `node` repl and test the `??` operator:

```
$ node
> 1 ?? 2
1
```

And second, try to run hardhat directly without using `npx`:

```
$ ❯ ./node_modules/.bin/hardhat
888    888                      888 888               888
888    888                      888 888               888
888    888                      888 888               888
8888888888  8888b.  888d888 .d88888 88888b.   8888b.  888888
888    888     "88b 888P"  d88" 888 888 "88b     "88b 888
888    888 .d888888 888    888  888 888  888 .d888888 888
888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888
```

**Comment by itsmeashim** [comment, marked as answer]
The folder i did this was non-empty. so i deleted everything and made a new folder and installed and did npx hardhat. it worked! idk if this thing solved it or it was something other. but it worked at last.

**Reply by fvictorio** [reply]
My guess is that you were accidentally using an older version of node. Sometimes that can happen if you are using some node version manager like nvm or volta.
